templates/definition/meter/smartfox.yaml

```diff
@@ -0,0 +1,72 @@
+template: smartfox
+products:
+  - brand: Smartfox
+    description:
+      generic: Smartfox Reg/Reg extended/Pro/Pro 2/Pro Light/Pro Light 2/Light
+params:
+  - name: usage
+    choice: ["grid", "pv"]
+  - name: host
+render: |
+  {{- define "uri" -}}
+  http://{{ .host }}/all
+  {{- end }}
+  type: custom
+  {{- if not .usage }}
```

Comment:
Wenn kein usage definiert ist, dann power von Heizstab für die Warmwasserbereitung zurück geben, so kann man es in "aux" mode nützen.